.github/actions/archive-repo/action.yaml

```diff
@@ -29,10 +29,10 @@ inputs:
     default: "us-east-1"
 
   bucket:
-    description: "Name of the S3 bucket to store the archive in"
+    description: "Name of the S3 bucket (and folder path if desired) to store the archive in"
     type: string
     required: false
-    default: "phil-code-archives"
+    default: "philrx-archives-stg/repositories"
```

Comment:
I don't think this is the bucket where we want to put this

Reply:
This was a change requested by @JwalantaPhil. Does the bucket have a specific purpose beyond just being a place for archived objects?

Reply:
He suggested putting it in a prod archive bucket or just leaving it in `phil-code-archives`. Can we just leave it in `phil-code-archives` please?